tests/acceptance/PaymentCept.php

```diff
@@ -71,12 +68,12 @@
 
 $I->amGoingTo('send third payment');
 $I->click('//a[contains(@class, \'ui--sendEmail\')]');
-$I->waitForElement('//*[@title="Odeslané emaily"]');
+//$I->waitForElement('//*[@title="Odeslané emaily"]');
 
 $page->seeNumberOfPaymentsWithState('Nezaplacena', 2);
 $page->seeNumberOfPaymentsWithState('Dokončena', 1);
 
-$I->seeEmailCount(1);
+//$I->seeEmailCount(1);
```

Comment:
Tady by stačilo minimálně checknout, jestli v DOM jsou ty informace o odeslání. Na e2e odeslání mailů máme jiný test, ale ta kontrola je tady poteba.

Reply:
To nevím jak udělat

Reply:
Např. těm [položkám, které existují pro každý odeslaný email](https://github.com/skaut/Skautske-hospodareni/blob/7bfa911ac0ca8e12abfc53f9c9ca88035f43ec45/app/AccountancyModule/PaymentModule/components/templates/PaymentList.grid.latte#L19) přidat třídu `ui--sentEmail` a pak kontrolovat pomocí `seeNumberOfElementsInDom()`

![image](https://user-images.githubusercontent.com/5658260/96431449-ce48a000-1203-11eb-9090-923f09237272.png)

Tedy např.

```php
$I->seeNumberOfElementsInDOM('.ui--sentEmail', 1);
```

Dvě poznámky:
- ten název třídy může být libovolný, ale měl by začínat prefixem `ui--`, který používáme na dalších místech, kdy se s takovými elementy interaguje v akceptačních testech
- je potřeba použít `seeNumberOfElementsInDOM`, nikoliv `seeNumberOfElements` - ten AFAIK bere jen aktuálně viditelné elementy, což položka v zavřeném dropdownu není.

Píšu z hlavy. Snad by to mělo fungovat :slightly_smiling_face:

Reply:
Jo, ve výsledku po opravě https://github.com/skaut/Skautske-hospodareni/pull/1427#discussion_r507649917, tohle asi fakt lze dropnout.

Reply:
ok, tedy to neresim